main/webapp/modules/core/styles/project/facets.less

```diff
@@ -185,15 +185,21 @@ li.facet-state-minimize a.facet-title-minimize:hover {
   font-weight: bold;
   color: #ff6a00;
   }
+  
 .facet-mode-inverted .facet-choice-selected .facet-choice-label {
   text-decoration: line-through;
   color: black;
   }
+  
 .facet-choice-link.facet-mode-inverted {
   text-decoration: line-through;
   color: black;
   }
 
+div.facet-title span.facet-title-span {
+  cursor: pointer;
```

Comment:
I would rather use `cursor: text` to better suggest the editing.

Reply:
@wetneb: here, I would respectfully disagree. The text cursor must be shown when the edit field is present/accessible. This will happen just after the dialog is opened and the cursor above the edit field.

In the next itineration of this feature, when we will be able to edit directly, then, the text cursor will be the best choice, IMOO.

Does it make sense?

Regards,
   Antoine

Reply:
If you want to keep a more "clickable" look and feel, then I would add a little pencil icon at the end of the current name when hovering it - that would suggest editing more visually, people would be able to understand it without reading the help text.

Reply:
> I would add a little pencil icon at the end of the current name when hovering it - that would suggest editing more visually, people would be able to understand it without reading the help text.

Do we currently have this behavior somewhere in OR that I could mimick/copy?

Regards,
   Antoine

Reply:
@antoine2711 I think he means using a custom image with an Pencil icon and using it for the CSS value for a custom cursor here:  https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Additionally perhaps with a keyword fallback for `text` or `pointer` where both are typically used to give indication to users that there is a clickable Link.

Something like this:
```css
.custom {
  cursor: url(images/edit_pencil.png), text;
}
```